src/main/java/org/opentripplanner/apis/gtfs/datafetchers/TripOccupancyImpl.java

```diff
@@ -0,0 +1,40 @@
+package org.opentripplanner.apis.gtfs.datafetchers;
+
+import static org.opentripplanner.apis.gtfs.generated.GraphQLTypes.GraphQLOccupancyStatus.CRUSHED_STANDING_ROOM_ONLY;
+import static org.opentripplanner.apis.gtfs.generated.GraphQLTypes.GraphQLOccupancyStatus.EMPTY;
+import static org.opentripplanner.apis.gtfs.generated.GraphQLTypes.GraphQLOccupancyStatus.FEW_SEATS_AVAILABLE;
+import static org.opentripplanner.apis.gtfs.generated.GraphQLTypes.GraphQLOccupancyStatus.FULL;
+import static org.opentripplanner.apis.gtfs.generated.GraphQLTypes.GraphQLOccupancyStatus.MANY_SEATS_AVAILABLE;
+import static org.opentripplanner.apis.gtfs.generated.GraphQLTypes.GraphQLOccupancyStatus.NOT_ACCEPTING_PASSENGERS;
+import static org.opentripplanner.apis.gtfs.generated.GraphQLTypes.GraphQLOccupancyStatus.NO_DATA_AVAILABLE;
+import static org.opentripplanner.apis.gtfs.generated.GraphQLTypes.GraphQLOccupancyStatus.STANDING_ROOM_ONLY;
+
+import graphql.schema.DataFetcher;
+import graphql.schema.DataFetchingEnvironment;
+import org.opentripplanner.apis.gtfs.generated.GraphQLDataFetchers;
+import org.opentripplanner.apis.gtfs.generated.GraphQLTypes;
+import org.opentripplanner.apis.gtfs.model.TripOccupancy;
+
+public class TripOccupancyImpl implements GraphQLDataFetchers.GraphQLTripOccupancy {
+
+  @Override
+  public DataFetcher<GraphQLTypes.GraphQLOccupancyStatus> occupancyStatus() {
+    return env -> {
+      var occupancyStatus = getSource(env).occupancyStatus();
+      return switch (occupancyStatus) {
+        case NO_DATA_AVAILABLE -> NO_DATA_AVAILABLE;
+        case EMPTY -> EMPTY;
+        case MANY_SEATS_AVAILABLE -> MANY_SEATS_AVAILABLE;
+        case FEW_SEATS_AVAILABLE -> FEW_SEATS_AVAILABLE;
+        case STANDING_ROOM_ONLY -> STANDING_ROOM_ONLY;
+        case CRUSHED_STANDING_ROOM_ONLY -> CRUSHED_STANDING_ROOM_ONLY;
+        case FULL -> FULL;
+        case NOT_ACCEPTING_PASSENGERS -> NOT_ACCEPTING_PASSENGERS;
```

Comment:
This code looks suspicious. Are you sure that the types are right?

Reply:
Honestly I had the same concerns but it seemed to work but it works even if it actually uses wrong types in this case... Maybe I should not use the short static import types here as this is a bit confusing as the same enum values come from two different enums?

Reply:
Ive changed this so that the return types can be distinguished.